components/upload/upload.component.ts

```diff
@@ -175,6 +179,8 @@ export class NzUploadComponent implements OnInit, OnChanges, OnDestroy {
   // #endregion
 
   constructor(
+    private ngZone: NgZone,
+    @Inject(DOCUMENT) private document: Document,
```

Comment:
```suggestion
    @Inject(DOCUMENT) private document: NzSafeAny,
```

建议使用 `NzSafeAny` 来代替，不然可能在 SSR 下会有问题。

Reply:
@cipchk 已按建议修改，谢谢~